src/tanks/Tank.js

```diff
@@ -84,9 +86,10 @@ class Tank extends GameObject {
 		this.chassis = chassis;
 		this.treads = treads;
 		this.mainGun = mainGun;
+		this.secondaryGun = secondaryGun;
 		this.scanner = scanner;
 		this.jammer = jammer;
-		this.parts = [chassis, treads, mainGun, scanner, jammer];
+		this.parts = [chassis, treads, mainGun, secondaryGun, scanner, jammer];
```

Comment:
Is this loading something different from what Adam had here?
>Parts will be an array of 11 values:
[0] = Chassis
[1] = Main Gun, [2] = Secondary Gun
[3] = Scanner, [4] = Scanner Addon 1, [5] = Scanner Addon 2
[6] = Jammer
[7] = Treads
[8] = Item 1, [9] = Item 2, [10] = Item 3

Reply:
I think Adam's array is the way things are stored on the back end. This is just the list of all of the tankparts used in the game. So I think they are different, yeah

Reply:
Alright that's all I was concerned about. Thanks for clarifying.